OasisDEX/oasis-borrow
# Ability to set approval and stop loss

# [Automation](https://app.shortcut.com/oazo-apps/story/2947/auto-implement-automation-approval-transaction)

### **Please do not comment on console logs, this is to feature branch, console.logs will be removed when i will be confident that all works just fine**
  
## Changes 👷‍♀️

- Added ability to send on Goerli transaction setting a trigger
- Added ability to set a stop loss level and choose type of stop loss
  
## How to test 🧪
 - Switch to goerli network
 - Create a new vault
 - Go to protection tab (Stop loss should have value '-')
 - set a stop loss and wait 
 - stop loss should set itself to proper value on the ui
 - edit stop loss again

**Code was tested with metamask, there might me malfunction with StopLoss refresh on UI for hardware wallets**
    
## Definition of done ✔️
